tensorflow_probability/python/bijectors/real_nvp_test.py

```diff
@@ -97,6 +98,67 @@ def testBatchedBijectorWithMLPTransform(self):
       self.assertAllClose(x_, inverse_y_, rtol=1e-1, atol=0.)
       self.assertAllClose(ildj_, -fldj_, rtol=1e-6, atol=0.)
 
+  def testBijectorConditionKwargs(self):
+    batch_size = 3
+    x_ = np.arange(batch_size * 4 * 2).astype(np.float64).reshape(
```

Comment:
@srvasude I had to change these to float64 in order to make the tests pass. Otherwise I would get the weird precision errors here:
```python
AssertionError:
Not equal to tolerance rtol=0.1, atol=0
Mismatched value: a is different from b.
(mismatch 4.166666666666671%)
 x: array([[ 0.,  1.,  2.,  3.,  4.,  5.,  6.,  7.],
       [ 8.,  9., 10., 11., 12., 13., 14., 15.],
       [16., 17., 18., 19., 20., 21., 22., 23.]], dtype=float32)
 y: array([[ 0.      ,  1.      ,  2.      ,  3.      ,  4.      ,  5.      ,
         5.999999,  7.      ],
       [ 8.      ,  9.      , 10.      , 11.      , 12.      , 13.000007,...

----------------------------------------------------------------------
Ran 18 tests in 6.292s
```

Reply:
why are we using atol = 0? seems unduly strict for floating point comparison...

Reply:
I happened to copy the tolerances from the tests above. The problem is though, that even when increasing this tolerance, my test case fails when not using float64. There are some errors on the relative scale of 50%.